extensions/amp-a4a/0.1/a4a-variable-source.js

```diff
@@ -23,7 +23,7 @@ import {
 } from '../../../src/service/variable-source';
 import {user, userAssert} from '../../../src/log';
 
-const WHITELISTED_VARIABLES = [
+const ALLOWED_VARIABLES = [
```

Comment:
We've been using `allowlist` elsewhere, can we be consistent with this name?

`const ALLOWLISTED_VARIABLES = [`